### DISCUSSION TITLE
Deleting an account

### DISCUSSION THREAD

**Discussion by ghost** [community/community#68924] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Body

Hi!
I want to delete my account by can't do that since the account is currently an owner in these organizations: tdorosh-org. But there is no such organization.
Pls help me to delete an account.
<img width="797" alt="image(3)" src="https://github.com/community/community/assets/43437210/9676bdef-605f-4eb5-ad62-752e40984460">
<img width="1716" alt="image(2)" src="https://github.com/community/community/assets/43437210/e9db1146-d6ae-406c-b3ec-ed02a7f7132c">
<img width="1725" alt="image(1)" src="https://github.com/community/community/assets/43437210/24b107b4-2ffa-4456-9d91-4d975d956bc6">

**Comment by 1amkaiz3n** [comment, marked as answer]
To delete your GitHub account, follow these steps:

1.Log In to GitHub:
Make sure you are logged into the GitHub account that you want to delete.

2.Go to Account Settings:
Click on your profile icon at the top right corner and select "Settings" from the dropdown menu.

3.Access Account Deletion:
In the left sidebar, scroll down and find the "Account settings" section. Click on "Account" in that section.

4.Initiate Account Deletion:
Scroll down to the bottom of the page, and you'll find the "Danger Zone." Click on the "Delete your account" link.

5.Verify Account Deletion:
GitHub will ask you to enter your password to confirm the deletion. Enter your password and click "Sign in."

6.Provide Reason and Verify Deletion:
You will be prompted to provide a reason for leaving. Select an appropriate reason and confirm your decision.

7.Finalize Deletion:
GitHub will ask you to verify the deletion by typing in your GitHub username. Enter your username and click "Delete account."

Please note that GitHub provides a grace period during which you can cancel the account deletion if you change your mind. After the grace period, your account and all associated data will be permanently deleted.

If you're encountering specific issues or errors during this process, I recommend reaching out to GitHub support for assistance. You can contact GitHub support through their [support page](https://support.github.com/contact) for account-related queries.

**Reply by ghost** [reply]
I did all that and it still won't let me!!!

**Comment by lbeauregard21** [comment]
Hi, I'm trying to delete my account but it doesn't work. Please delete my account : lbeauregard21. Thanks

**Reply by Alidarmeny** [reply]
مرحبًا، أحاول حذف حسابي ولكن الأمر لا ينجح. يُرجى حذف حسابي: lbeauregard21. شكرًا

**Comment by jacknev80-create** [comment]
I don't understand it and find it completely shit and would like to delete whatever the hell I'm signed up for

**Comment by ghost** [comment]
Just delete the account please I'd do.it myself but you guys have seem to make that impossible
